.github/workflows/blueprint.yml

```diff
@@ -48,29 +48,23 @@ jobs:
       - name: Get Mathlib cache
         run: ~/.elan/bin/lake exe cache get || true
 
-      - name: Build project
-        run: ~/.elan/bin/lake build equational_theories
-
-      - name: Cache mathlib API docs
+      - name: Cache build artifacts and Mathlib API docs
         uses: actions/cache@v4
         with:
           path: |
-            .lake/build/doc/Init
-            .lake/build/doc/Lake
-            .lake/build/doc/Lean
-            .lake/build/doc/Std
-            .lake/build/doc/Mathlib
-            .lake/build/doc/declarations
-            .lake/build/doc/find
-            .lake/build/doc/*.*
+            .lake/build
+            !.lake/build/doc-data
             !.lake/build/doc/declarations/declaration-data-equational_theories*
-          key: MathlibDoc-${{ hashFiles('lake-manifest.json') }}
-          restore-keys: MathlibDoc-
+          key: LakeBuild-${{ hashFiles('lake-manifest.json') }}
+          restore-keys: LakeBuild-
```

Comment:
GitHub action caching is very confusing here, but this will not work: the `key` needs to include the git ref, else it will never push a new cache if the lake manifest stays the same. 

Maybe follow this example:
https://github.com/leanprover/lean-action/blob/9e84bf82a65182a555b9c9b4b78500fa203dc5a4/action.yml#L136

Reply:
Done. Thanks.